tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingTestCase.java

```diff
@@ -150,6 +157,29 @@ public void testPull() {
         Assert.assertTrue(Files.exists(tempHomeDirectory.resolve(baloPath).resolve(baloFileName)));
     }
 
+    @Test(description = "Test pullCount of a package from central", dependsOnMethods = "testPull")
+    public void testPullCount() throws IOException {
+        URI remoteUri = URI.create(RepoUtils.getRemoteRepoURL() + "/modules/info/" + orgName + "/" + moduleName);
+        HttpURLConnection conn;
+        conn = (HttpURLConnection) remoteUri.toURL().openConnection();
+        int statusCode = conn.getResponseCode();
+        if (statusCode == 200) {
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(conn.getInputStream(),
+                    Charset.defaultCharset()))) {
+                StringBuilder result = new StringBuilder();
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    result.append(line);
+                }
+                Object payload = JSONParser.parse(result.toString());
+                if (payload instanceof MapValue) {
+                    long pullCount = ((MapValue) payload).getIntValue("totalPullCount");
+                    Assert.assertEquals(pullCount, 1);
+                }
+            }
+        }
+    }
+
 //    @Test(description = "Test searching a package from central", dependsOnMethods = "testPush")
```

Comment:
Please check why this test is disabled by commenting out. Better to use `enabled  = false` otherwise this never get fixed as it won't capture for the `enabled  = false` search criteria.

Reply:
This test got passed. As this is already commented, I will give a separate PR with uncommenting this test.